config/routes.rb

```diff
@@ -3,4 +3,16 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # get '/dishes/:id', to: 'dishes#show'
+  resources :dishes, only: [:show] do
+    resources :ingredients, controller: 'dish_ingredients', only: [:create, :destroy]
+  end
+
+  resources :chef, controller: 'chefs', only: [:show] do
+    resources :ingredients, controller: 'chef_ingredients', only: [:index]
+  end
```

Comment:
Nice use of advanced routing! Just FYI that it's totally okay to handroll a route on an IC if you prefer. Either works, but this is great practice with the new syntax!